web/config/config.go

```diff
@@ -0,0 +1,92 @@
+package config
+
+import (
+	"os"
+
+	"github.com/autograde/quickfeed/internal/rand"
+)
+
+const (
```

Comment:
Maybe these `consts` could be organized into separate `const` sections or in separate files.

Reply:
Moved to a separate file.